src/libraries/Directory.Build.targets

```diff
@@ -39,8 +39,6 @@
     <NoWarn Condition="'$(IsPartialFacadeAssembly)' == 'true'">$(NoWarn);SYSLIB0003;SYSLIB0004;SYSLIB0015;SYSLIB0017;SYSLIB0021;SYSLIB0022;SYSLIB0023;SYSLIB0025;SYSLIB0032</NoWarn>
     <!-- Reset these properties back to blank, since they are defaulted by Microsoft.NET.Sdk -->
     <WarningsAsErrors Condition="'$(WarningsAsErrors)' == 'NU1605'" />
-    <!-- Set the documentation output file globally. -->
-    <DocumentationFile Condition="'$(IsSourceProject)' == 'true' and '$(DocumentationFile)' == '' and '$(MSBuildProjectExtension)' != '.vbproj'">$(OutputPath)$(TargetName).xml</DocumentationFile>
```

Comment:
Why is this no longer needed? Is this the default location for the compiler?

Reply:
The sdk handles that for us: https://github.com/dotnet/sdk/blob/e9d4ebb83d44fb21c910a1fbcaca6bbfd6bf8630/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets#L247-L260. No need to set the path manually ourselves.